### DISCUSSION TITLE
Access applied changes in Plandex Cloud

### DISCUSSION THREAD

**Discussion by jzy95310** [plandex-ai/plandex#2] [General]
Hi sir,

Thank you for sharing this amazing programming engine in the AI community! I gave it a try yesterday to ask it to generate a new code snippet based on the given instructions. However, after I committed the changes, I did not see anywhere to access the newly generated code. Am I expected to access it on Plandex Cloud, and if so, how can I access it through the terminal?

**Comment by danenania** [comment]
Hey @jzy95310, thanks for trying it out! After running `plandex apply` to apply the changes, the new code should then be added to your project files. Are you not seeing the changes in your project?

You can also run `plandex convo` to see the full history of the conversation, including the code that was suggested.

If you want to undo the `plandex apply` command so that you can see the changes in `plandex changes` again, you can use `plandex rewind` to revert to before the `plandex apply` command (use `plandex log` to check the history before rewinding). I should note that `plandex rewind` won't impact your project files in any way.

Does that help at all?

**Comment by jzy95310** [comment]
Hi @danenania, thank you so much for your guidance. I might have accidentally run `plandex rewind` so the changes are canceled. I now see the newly generated files.

**Comment by danenania** [comment]
@jzy95310 Great!

One thing you could try in the future to avoid losing anything is to checkout a new branch first before rewinding with `plandex checkout`. Then you can rewind in the new branch, but still keep everything you've done in the `main` branch in case you want to go back to it.

I'm also interested in adding a feature like `git reflog` at some point that would allow undoing a `rewind` and returning to any previous state.

Btw I'm curious if you have any feedback so far?

**Comment by jzy95310** [comment]
@danenania I think the plandex should be pretty useful for managing code snippets and files efficiently in batches. However, I have not had a chance to run more comprehensive tests. Thanks for your contribution to developing this amazing programming engine!

**Reply by danenania** [reply]
Thanks @jzy95310 -- I moved this over to a discussion. Please feel free to add feedback as you do more testing!
